qa/1795

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+# PCP QA Test No. 1795
+# Exercise the BPF PMDA CO-RE vfsstat module - install, remove and values.
+#
+# Copyright (c) 2022 Sohaib Mohamed.
+#
+
+seq=`basename $0`
+echo "QA output created by $seq"
+
+. ./common.bpf
+
+_pmdabpf_check
+_pmdabpf_require_kernel_version 5 0
+_pmdabpf_require_libbpf_version 0 7
+
+status=1       # failure is the default!
+signal=$PCP_BINADM_DIR/pmsignal
+$sudo rm -rf $tmp.* $seq.full
+
+_cleanup()
+{
+    _pmdabpf_cleanup
+}
+
+_prepare_pmda bpf
+trap "_cleanup; exit \$status" 0 1 2 3 15
+_stop_auto_restart pmcd
+
+# real QA test starts here
+cat <<EOF | _pmdabpf_install
+# Installed by PCP QA test $seq on `date`
+[vfsstat.so]
+enabled=true
+EOF
+_pmdabpf_wait_for_metric
+
+# Generate system activity for the CO-RE vfsstat module
```

Comment:
we should sleep some time here, or do something?

Reply:
Done! Thanks
```
$ git diff
diff --git a/qa/1795 b/qa/1795
index 44752897d..68e5c7818 100755
--- a/qa/1795
+++ b/qa/1795
@@ -35,6 +35,9 @@ enabled=true
 EOF
 _pmdabpf_wait_for_metric

+# Wait for values to appear
+pmsleep 2
+
```